Libraries/WebSocket/RCTWebSocketModule.m

```diff
@@ -99,6 +100,27 @@ - (void)dealloc
   [_sockets[socketID] send:message];
 }
 
+RCT_EXPORT_METHOD(sendBlob:(NSDictionary *)blob socketID:(nonnull NSNumber *)socketID)
+{
+  RCTBlobManager *blobManager = [[self bridge] moduleForClass:[RCTBlobManager class]];
```

Comment:
I think we should revisit this: it would be nice for this module to be completely optional, by linking it here, you're basically required to adopt this.